package.json

```diff
@@ -40,11 +40,10 @@
     "gulp-plumber": "^1.0.0",
     "isparta": "^3.0.3",
     "mongoose": "^4.2.5",
-    "sinon": "1",
-    "sinon-as-promised": "^4.0.0"
+    "sinon": "^2.1.0"
   },
   "peerDependencies": {
-    "sinon": "1"
+    "sinon": "^2.1.0"
```

Comment:
If we do that, then there needs to be a workaround to have sinon-as-promised -dependency. I actually didn't tested if actual code fix works on sinon 1 :). 

Probably would be easier to release version 2 of sinon-mongoose which supports only Sinon 2. Then have a documentation instructing use version 1.3.0 sinon-mongoose if working with Sinon 1.